blog/_posts/2022-01-17-scala-3.1.1-released.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.1 and forward compatibility news
+---
+
+Hello from the Scala 3 team! After a slightly longer than usual pause, we are back with a new release of the Scala compiler and exciting news about improvements in forward compatibility. The delay in the release was caused by urgent bugs that needed backporting and testing, and then the holiday season.
+
+## Improvements is Scala 3.1.1
+
+Scala 3.1.1-RC2 was promoted to a full stable release of the language. Scala 3.1.1 is the first patch version after the 3.1 minor release. It is therefore focused on bug fixes and improvements in overall correctness and stability. Although there are no new additions to the language, you can still spot some noticeable improvements:
+
+- You can use `using` clauses to introduce Scala 2 conversions (`<:<` operator).
```

Comment:
Maybe add links to PRs?

Reply:
Agree this needs further explanation, perhaps just in the form of a link; I cannot make sense of the description